lib/default-theme/SearchBox.vue

```diff
@@ -184,17 +184,40 @@ export default {
       a
         color $accentColor
 
-@media (max-width: $MQNarrow)
+@media (max-width: $MQMobile)
   .search-box
+    margin-right 0
+    display inline-block
     input
+      position relative
+      left 1rem
       cursor pointer
       width 0
       border-color transparent
-      position relative
-      left 1rem
+      background-color white
+      z-index 10
+      &:focus
+        cursor text
+        width 10rem
+
+@media (min-width: $MQMobile) and (max-width: $MQNarrow)
```

Comment:
Currently there are multiple media queries with duplicated rules that can be merged together. e.g. This is the exact same media query as the one on line 223.